app/ui.js

```diff
@@ -1643,12 +1640,12 @@ const UI = {
 const LINGUAS = ["de", "el", "es", "nl", "pl", "sv", "zh"];
 l10n.setup(LINGUAS);
 if (l10n.language !== "en" && l10n.dictionary === undefined) {
-    WebUtil.fetchJSON('app/locale/' + l10n.language + '.json', function (translations) {
+    WebUtil.fetchJSON('app/locale/' + l10n.language + '.json', (translations) => {
         l10n.dictionary = translations;
 
         // wait for translations to load before loading the UI
         UI.prime();
-    }, function (err) {
+    }, (err) => {
```

Comment:
Should be able to skip the '{' and '}' here right?

Reply:
yup. But this change was just changed on master